legal-api/src/legal_api/reports/report.py

```diff
@@ -356,12 +356,12 @@ def _set_dates(self, filing):
         filing["effective_date_time"] = LegislationDatetime.format_as_report_string(effective_date)
         filing["effective_date"] = effective_date.strftime(OUTPUT_DATE_FORMAT)
         # Recognition Date
-        if self._legal_entity:
-            recognition_datetime = LegislationDatetime.as_legislation_timezone(self._legal_entity.founding_date)
+        if self._business:
+            recognition_datetime = LegislationDatetime.as_legislation_timezone(self._business.founding_date if self._business.is_legal_entity else self._business.start_date)
```

Comment:
Are we okay with making `recognition_date` for alternate name entities be the `start_date`, since there is no `founding_date`? Or do we just not want to include the recognition date in the output for alternate names?